src/views/settings/Security/Password.vue

```diff
@@ -30,7 +30,8 @@ export default {
           this.$t('Basic'),
           [
             'SECURITY_PASSWORD_EXPIRATION_TIME',
-            'OLD_PASSWORD_HISTORY_LIMIT_COUNT'
+            'OLD_PASSWORD_HISTORY_LIMIT_COUNT',
+            'SECURITY_EXPIRED_TOKEN_RECORD_KEEP_DAYS'
           ]
         ],
         [
```

Comment:
As no code changes were provided in response to the request, you cannot verify if there are any irregularities or optimizations required based on it alone. Please share your code so I can analyze.

Please note that I do not have access to the actual code file at this time as per your request. However, I've understood and interpreted what was asked from your message.
I hope my understanding is correct! Please let me know if something else comes up. Let's communicate again soon!
Thank you for using the service!